<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="m_populationLabel">
     <property name="text">
      <string>Население</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_prefixLineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_typeComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="m_lonLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="m_nameEngLineEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="m_countryLineEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="m_mapPointLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="m_latLabel">
     <property name="text">
      <string>Широта</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_acceptButton">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancelButton">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="m_prefixLabel">
     <property name="text">
      <string>Префикс</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="m_countryLabel">
     <property name="text">
      <string>Страна</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="m_nameLabel">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="m_descriptionLabel">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="m_lonLabel">
     <property name="text">
      <string>Долгота</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="m_populationLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_nameEng">
     <property name="text">
      <string>Название(Английский)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_typeLabel">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="m_mapPointLabel">
     <property name="text">
      <string>Точка привязки</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="m_descriptionLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_nameLineEdit"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="m_latLineEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="m_altLabel">
     <property name="text">
      <string>Высота</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="m_altLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
